task_12/index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="theme-color" content="#e65c00">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link rel="stylesheet" href="style/css/main.css">
+    <link rel="icon" type="image/vnd.microsoft.icon" href="img/favicon.ico">
+    <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+    <title>Pizza App</title>
+</head>
+<body>
+    <header>
+        <div class="time">
+            <span>
+                <i class="far fa-clock"></i>
+                <span id="time">00:00:00</span>
```

Comment:
Please, employ semantic element `<time>` with `datetime` attribute for better accessibility.

Reply:
Thanks for idea, <span> element replaced with <time>